auditwheel/policy/policy.json

```diff
@@ -25,5 +25,26 @@
          "libX11.so.6", "libXext.so.6", "libXrender.so.1", "libICE.so.6",
          "libSM.so.6", "libGL.so.1", "libgobject-2.0.so.0",
          "libgthread-2.0.so.0", "libglib-2.0.so.0", "libresolv.so.2"
+     ]},
+    {"name": "manylinux2010",
+     "priority": 200,
```

Comment:
Following up on my most recent comment, I believe this priority should be lowered such that it is between manylinux1 (most restrictive) and "linux" (least restrictive). Current manylinux1 binaries should also be manylinux2010 compliant, but not vice versa.

Reply:
Perhaps we should have done the priorities in increasing rather than decreasing order... feel free to pick an arbitrary number between 0 and 100 for now (90?) and if we need to renumber later because the range was too small we can do that eventually.